packages/core/botpress/src/web/components/App/index.jsx

```diff
@@ -49,6 +53,22 @@ class App extends Component {
     EventBus.default.on('notifications.new', notification => {
       this.props.addNotifications([notification])
     })
+
+    EventBus.default.on('toast.nlu-sync', ({ text, type, time = 5000 }) => {
+      if (this.toastNLUId) {
+        toast.dismiss(this.toastNLUId)
+      }
+
+      this.toastNLUId = toast[type](text, { autoClose: time })
+    })
+
+    EventBus.default.on('toast.qna-save', ({ text, type, time = 5000 }) => {
+      if (this.toastQnAId) {
+        toast.dismiss(this.toastQnAId)
+      }
+
+      this.toastQnAId = toast[type](text, { autoClose: time })
+    })
```

Comment:
@dmytropanontko , it seems [toast is available](https://github.com/botpress/botpress/commit/63c636162150008895506c88c70b0d8733d9b5fd) directly. Can we use it through `bp.toast`?

Reply:
@epaminond no

Reply:
Let's make API more generic here since this is a core.